dev-python/pymacaroons/pymacaroons-0.13.0-r1.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="PyMacaroons is a Python implementation of Macaroons."
+HOMEPAGE="
+	https://github.com/ecordell/pymacaroons
+	https://pypi.org/project/pymacaroons/
+"
+SRC_URI="
+	https://github.com/ecordell/pymacaroons/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+	https://github.com/arkamar/pymacaroons/commit/backport-upstream-pr59.patch
+		-> ${P}-nose-to-pytest.patch
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/pynacl[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+BDEPEND="test? (
```

```suggestion
BDEPEND="
	test? (
```

This is very unreadable if you jump indentation like that.